### DISCUSSION TITLE
Lut installation

### DISCUSSION THREAD

**Discussion by appleimperio** [awawa-dev/HyperHDR#825] [Q&A] [answered]
I just want to know if you install a LUT through the web interface the button will change from install to installed? Because I have an old LUT i hit install and I don't know if it replaced it or not. keep reading install even after the restart.

**Comment by awawa-dev** [comment, marked as answer]
No, the label wont change. But the CRC on the same page will change if you install another LUT.

**Reply by appleimperio** [reply]
Thank You!
